indexreport.go

```diff
@@ -7,6 +7,7 @@ package claircore
 //
 // A list of these can be thought of as an "unpacked" IndexReport
 type IndexRecord struct {
+	ID           string
```

Comment:
Why does the root IndexRecord grow an ID?
Each object within the IndexRecord has an ID.